<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestao_remover</class>
 <widget class="QDialog" name="gestao_remover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color:#000;
}


QLineEdit{
background-color:#fff;
color:#000;
padding:10px;
border-radius:10px;
borde:1px solid #fff;
}

QLabel{
color:#fff
}</string>
  </property>
  <widget class="QPushButton" name="btn_gr_pesquisar">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>32</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pesquisar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_gr_mostraPesquisa">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>320</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#fff;</string>
   </property>
   <property name="title">
    <string>Metodo de pesquisa</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>60</width>
      <height>46</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QRadioButton" name="rb_gr_cod">
       <property name="styleSheet">
        <string notr="true">color:#fff</string>
       </property>
       <property name="text">
        <string>Código</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_gr_nome">
       <property name="styleSheet">
        <string notr="true">color:#fff</string>
       </property>
       <property name="text">
        <string>Nome</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLineEdit" name="txt_gr_pesquisa">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>59</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#000</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Código do produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_gr_codigo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Nome do produto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_gr_nome"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Quantidade em estoque</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_gr_qtde_estoque"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Quantidade a remover</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_gr_qtde_remover"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_gr_voltar">
        <property name="text">
         <string>Voltar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_gr_remove">
        <property name="text">
         <string>remover</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_txt_mostra_podutos">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>32</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> Todos os produtos</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
